lib/src/command/get.dart

```diff
@@ -28,6 +28,13 @@ class GetCommand extends PubCommand {
         negatable: false,
         help: "Report what dependencies would change but don't change any.");
 
+    argParser.addFlag(
+      'enforce-lockfile',
+      negatable: false,
+      help:
+          'Enforce pubspec.lock. Fail resolution if pubspec.lock does not satisfy pubspec.yaml',
```

Comment:
```suggestion
          'Enforce pubspec.lock, fails resolution if pubspec.lock does not satisfy pubspec.yaml',
```
Don't know, maybe I'm just crazy -- English is not my first language.

Reply:
I think both are valid. Landing as is - we can follow up with a fix if you disagree.